pandas/_libs/lib.pyx

```diff
@@ -26,6 +26,7 @@ from cpython.tuple cimport (
     PyTuple_SET_ITEM,
 )
 from cython cimport floating
+import datetime
```

Comment:
i see why you're doing it, but im not wild about adding to the namespace just for doctests

Reply:
I think you can add the import as part of the doctests?

Reply:
yeah, done that now